python/cucim/src/cucim/skimage/transform/tests/test_geometric.py

```diff
@@ -147,13 +147,17 @@ def test_similarity_estimation():
 
 
 def test_3d_similarity_estimation():
-    src_points = cp.random.rand(1000, 3)
+    # Note: some choices of seed will produce poorly conditioned values,
+    #       leading to NaN in the estimate. We fix the seed here to avoid this.
+    seed = 10
+    rng = cp.random.default_rng(seed)
+    src_points = rng.random(size=(1000, 3))
```

Comment:
Maybe we should consider parameterizing these. Though that can be a follow up item separate from this PR